playwright/src/main/java/com/microsoft/playwright/Frame.java

```diff
@@ -1410,80 +1412,64 @@ public IsEnabledOptions setTimeout(double timeout) {
   }
   class IsHiddenOptions {
     /**
-     * When true, the call requires selector to resolve to a single element. If given selector resolves to more than one
+     * When true, the call requires selector to resolve to a single element. If given selector resolves to more↵than one
      * element, the call throws an exception.
      */
     public Boolean strict;
-    /**
-     * **DEPRECATED** This option is ignored. {@link Frame#isHidden Frame.isHidden()} does not wait for the element to become
```

Comment:
you don't render deprecated method reasons anymore, its now deprecated: string instead of deprecated: bool and contains the reason.